pkgs/top-level/all-packages.nix

```diff
@@ -10836,7 +10836,8 @@ in
   })
     ruby_2_5
```

Comment:
Can we drop 2.5 anytime soon?

Reply:
By March end after it hits EOL, maybe: https://www.ruby-lang.org/en/downloads/branches/

Reply:
Since we know it's EOL'd before the next release, it can be done ASAP for unstable. Though I suggest in another PR.

Reply:
I didn't see that anyone else had done it, so: #111662.